learn/execution.md

```diff
@@ -0,0 +1,40 @@
+# Execution Layers in Rollkit
+
+Rollkit is designed to be modular and flexible, allowing different execution layers to be plugged in via the ABCI (Application Blockchain Interface) protocol. Rollkit defines a general-purpose execution interface ([see execution.go](https://github.com/rollkit/rollkit/blob/main/core/execution/execution.go)) that enables developers to integrate any compatible application as the rollup's execution layer.
```

Comment:
its not via the ABCI protocol, abci is a execution environment for cosmos sdk. the next line is right though.